src/libraries/Microsoft.PowerFx.Core/Public/ExpressionError.cs

```diff
@@ -170,4 +170,18 @@ internal static IEnumerable<ExpressionError> New(IEnumerable<IDocumentError> err
             }
         }
     }
+
+    internal class ExpressionErrorComparer : EqualityComparer<ExpressionError>
+    {
+        // We compare only Message
+        public override bool Equals(ExpressionError error1, ExpressionError error2)
+        {
+            return error1.ToString() == error2.ToString();
+        }
+
+        public override int GetHashCode(ExpressionError error)
+        {
+            return 0;
```

Comment:
> return 0; [](http://example.com/codeflow?start=11&length=10)

Should either thow or generate unique hash code.

Reply:
This class is specific for comparing error messages. We could create a new one in case we need to work with `Dict` and substitute this one.

Reply:
I feel we should probably return `error.ToString().GetHashCode()` or throw not implemented here...